js/apps/system/_api/foxx/APP/index.js

```diff
@@ -409,6 +409,9 @@ instanceRouter.post('/tests', (req, res) => {
 
 instanceRouter.get('/bundle', (req, res) => {
   const service = req.service;
+  if (service.isDevelopment) {
+    FoxxManager._createServiceBundle(service.mount);
```

Comment:
@mpv1989 I'm no longer sure this is a good idea, you probably told me not to do this earlier.

For the public API it would be helpful to get a bundle for the current state of the service when it is in development mode but simply overwriting the bundle in place will likely wreak havoc during the cluster self-healing (i.e. local changes might be considered evidence of bad state and "fixed" by the Foxxmaster). I'm not sure how to reconcile these two requirements.

Maybe we need to move this route into `_local` and just provide an equivalent to Aardvark's download route in the public API (i.e. create an ad-hoc zip bundle in a temp folder).